runtime/jcl/unix/syshelp.c

```diff
@@ -241,7 +241,7 @@ void mapLibraryToPlatformName(const char *inPath, char *outPath) {
 #else
 	strcpy(outPath, "lib");
 	strcat(outPath,inPath);
-#if defined(AIXPPC) && (JAVA_SPEC_VERSION == 8)
+#if defined(AIXPPC) && (JAVA_SPEC_VERSION <= 11)
```

Comment:
I don't think we want to change 14 at this point.  Based on my understanding, 15 is the earliest release we may have to change if the `alternativeLibraryName` doesn't work.

Let's remove the version check for the 0.21.0 release and address in the jdk15 release.  If the check is needed for 15 to build, then we can make it `<= 14` for the time being

Reply:
Agreed that `JDK 14` should be same as `JDK 11/8`. `JDK15` is the earliest release requiring the change as per https://github.com/eclipse/openj9/pull/8941, otherwise following `UnsatisfiedLinkError` will be thrown during `JDK 15 AIX` build:
```
12:35:36  Exception in thread "main" java/lang/UnsatisfiedLinkError: sun/nio/fs/UnixNativeDispatcher.init()I
12:35:36  	at sun/nio/fs/UnixNativeDispatcher.<clinit> (java.base@9/UnixNativeDispatcher.java:654)
12:35:36  	at sun/nio/fs/UnixFileSystem.<init> (java.base@9/UnixFileSystem.java:65)
```
Making it `<= 14` for the time being.

@DanHeidinga it is ready for another look.